submitter/sqs.py

```diff
@@ -10,9 +10,19 @@
 logger = logging.getLogger(__name__)
```

Comment:
I'd like to refactor this module at some point because I think we could make it cleaner, but for now 👍 .